src/criteria.py

```diff
@@ -0,0 +1,51 @@
+from symptom import Symptom
+from signal_statistics import SignalStatistics
+import constants
+
+""" Module for defining criteria to be used to evaluate event groups"""
```

Comment:
I probably missed this on the prior PRs, but module docstrings should be at the very top of the module, even head of the imports.